src/Patches/GhostHints.cs

```diff
@@ -287,6 +286,8 @@ public class HintGhost {
 
         public static void GenerateHints() {
             HintGhosts.Clear();
+            if (SaveFile.GetInt(EntranceRando) == 1 && !GhostLocations.ContainsKey("Purgatory"))
+            { GhostLocations.Add("Purgatory", new List<HintGhost> { new HintGhost("Purgatory", new Vector3(27.1514f, 38.018f, 74.7217f), new Quaternion(0f, 0.9585385f, 0f, -0.2849632f), NPC.NPCAnimState.DANCE, $"doo yoo nO skipEO? hE brOk awl uhv #uh dorz.") }); }
             List<string> GhostSpawns = GhostLocations.Keys.ToList();
```

Comment:
```suggestion
            List<string> GhostSpawns = GhostLocations.Keys.ToList();
            if (SaveFile.GetInt(EntranceRando) == 0) {
                GhostSpawns.Remove("Purgatory");
            }
```
Add the Purgatory entry back to the static GhostLocations dictionary and remove the other logic above this